openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/SystemRepository.java

```diff
@@ -206,6 +217,27 @@ public void updateSetting(Settings setting) {
     }
   }
 
+  public static <T> void validateJsonSchema(Object fromValue, Class<T> toValueType) {
+    // Convert JSON to Java object
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
use the one in Jsonutil, creating this on the fly can impact performance

Reply:
we will need the newly created validateJsonSchema method  to handle and validate complex JSON data that includes nested JSON objects, existing JsonUtils methods do not support nested structures 

moved the method to JsonUtils